docs/source/en/model_doc/flan-t5.mdx

```diff
@@ -0,0 +1,43 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# FLAN-T5
+
+## Overview
+
+FLAN-T5 was released in the paper [Scaling Instruction-Finetuned Language Models](https://arxiv.org/pdf/2210.11416.pdf) - it is an enhanced version of T5 that has been finetuned in a mixture of tasks.
+
+One can directly plug in the weights of FLAN-T5 into a T5 model, like so:
+
+```python
+>>> from transformers import T5ForConditionalGeneration
```

Comment:
I'd advertise the auto model class, since it works for any checkpoint.

Reply:
Added more content in ffabbb1ecd8719fa1da2eb6bf6b4ca1366b1707e